tests-system/lobster-cpptest/rbt-valid-source-input-file-paths/one-valid-file-scenario/expected-output/stdout.txt

```diff
@@ -0,0 +1,587 @@
+Written 5 lobster items to "cpptest.lobster".
+{
+  "data": [
+    {
+      "tag": "cpp test_case.cpp:TestMacrosTest:3",
+      "location": {
+        "kind": "file",
+        "file": "..\\..\\..\\data\\test_case.cpp",
```

Comment:
Please use Unix-style forward slashes `/` instead of Windows-style `\\`
That is the reason why the test suites are failing (I guess).
Note: The system test framework translates `\\` to `/`, to support running tests on Windows.